packages/app-desktop/gui/ResourceScreen.tsx

```diff
@@ -84,7 +84,13 @@ const ResourceTableComp = (props: ResourceTable) => {
 	};
 
 	const filteredResources = props.resources.filter(
-		(resource: InnerResource) => !props.filter || resource.title?.includes(props.filter) || resource.id.includes(props.filter),
+		(resource: InnerResource) => {
+			if (!props.filter) {
+				const filterLowerCase = props.filter.toLowerCase();
```

Comment:
```suggestion
			if (props.filter) {
				const filterLowerCase = props.filter.toLowerCase();
```
Please double-check this logic: If `!props.filter`, then `props.filter.toLowerCase()` will either fail (if `props.filter` is undefined) or return `''` (if `props.filter` is `''`).

Reply:
You're right, thanks for pointing that out!